src/modules/logger/messages.h

```diff
@@ -88,19 +88,41 @@ struct ulog_key_header_s {
 	uint8_t	data[0];
 };
 
-#define ULOG_MSG_HEADER_LEN 3 //accounts for msg_size and msg_type
+
+/**
+ * @brief Message Header for the uLog
```

Comment:
Below as well
```suggestion
 * @brief Message Header for the ULog
```

Reply:
What do you mean? Capital case? I always tend to stick to uLog as it comes from the uORB I guess, but ULog is also fine 🤔

Reply:
Not that it matters much to me, but it's generally written as ULog.